pkg/account/wallet.go

```diff
@@ -0,0 +1,162 @@
+/*
+Copyright 2023 The Bestchains Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package account
+
+import (
+	"crypto/x509"
+	"encoding/json"
+	"encoding/pem"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+type IWallet interface {
+	StoreAccount(*Account) error
+	GetAccount(string) (*Account, error)
+	ListAccounts() ([]string, error)
+	DeleteAccounts(...string) error
+}
+
+var _ IWallet = new(LocalWallet)
```

Comment:
as for "Verify Interface Compliance", it is more common to use the statement like `var _ http.Handler = (*Handler)(nil)`, see https://github.com/uber-go/guide/blob/master/style.md#verify-interface-compliance

Reply:
great! let's use same statement on any other `verify interface`